index.js

```diff
@@ -14,6 +14,7 @@
  * @private
  */
 
+var nodeStream = require('node:stream')
```

Comment:
We can do it this way
```suggestion
var { finished } = require('node:stream')
```